mp/src/game/server/neo/neo_game_config.cpp

```diff
@@ -7,6 +7,11 @@ LINK_ENTITY_TO_CLASS(neo_game_config, CNEOGameConfig);
 
 BEGIN_DATADESC(CNEOGameConfig)
 	DEFINE_KEYFIELD(m_GameType, FIELD_INTEGER, "GameType"),
+	DEFINE_KEYFIELD(m_HiddenHudElements, FIELD_INTEGER, "HiddenHudElements"),
```

Comment:
```suggestion
	DEFINE_KEYFIELD(m_HiddenHudElements, FIELD_INTEGER, "spawnflags"),
```
This isn't how it's usually implemented but if we do this we can use the flags menu in Hammer instead of having mappers input a random number into the KV. Tested and it doesn't appear to cause any issues
![image](https://github.com/user-attachments/assets/d17818d3-f824-401d-8c8e-5c273bef0ab5)